core/database.py

```
@@ -0,0 +1,43 @@
+DB_USER = os.getenv("POSTGRES_USER")
+DB_PASSWORD = os.getenv("POSTGRES_PASSWORD")
+DB_NAME = os.getenv("POSTGRES_DB")
+SUPABASE_PROJECT_REF = os.getenv("SUPABASE_PROJECT_REF")
+
+# Verifica se todas as variáveis necessárias estão presentes
+if not all([DB_USER, DB_PASSWORD, SUPABASE_PROJECT_REF, DB_NAME]):
+    raise ValueError("Por favor, configure POSTGRES_USER, POSTGRES_PASSWORD, SUPABASE_PROJECT_REF e POSTGRES_DB no seu arquivo .env")
+
+# --- Conexão via Transaction Pooler (Compatível com IPv4/Docker) ---
+# Hostname estático para o pooler da região sa-east-1
+DB_HOST = f"aws-0-sa-east-1.pooler.supabase.com"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)